047-TrafficControlWithDapr/README.md

```diff
@@ -0,0 +1,73 @@
+
+## Challenges
+- Challenge 0: **[Install local tools and Azure prerequisites](Student/Challenge-00.md)**
+   - Install the prerequisites tools and software as well as create the Azure resources required.
+- Challenge 1: **[Run the sample application](Student/Challenge-01.md)**
+   - Run the Traffic Control application to make sure everything works correctly.
+- Challenge 2: **[Dapr Service Invocation](Student/Challenge-02.md)**
+   - Add Dapr into the mix, using the Dapr service invocation building block.
+- Challenge 3: **[Dapr Pub/Sub Messaging](Student/Challenge-03.md)**
+   - Add Dapr publish/subscribe messaging to send messages from the `TrafficControlService` to the `FineCollectionService`.
+- Challenge 4: **[Dapr Redis State Management](Student/Challenge-04.md)**
+   - Add Dapr state management in the `TrafficControlService` to store vehicle information.
+- Challenge 5: **[Dapr SMTP Output binding](Student/Challenge-05.md)**
+   - Use a Dapr output binding in the `FineCollectionService` to send an email.
+- Challenge 6: **[Dapr MQTT Input Binding](Student/Challenge-06.md)**
+   - Add a Dapr input binding in the `TrafficControlService`. It'll receive entry- and exit-cam messages over the MQTT protocol.
+- Challenge 7: **[Dapr Secrets Management](Student/Challenge-07.md)**
+   - Add the Dapr secrets management building block.
+- Challenge 8: **[Dapr-enabled Services running in Azure Kubernetes Service (AKS)](Student/Challenge-08.md)**
+   - Deploy the Dapr-enabled services you have written locally to an Azure Kubernetes Service (AKS) cluster.
+
+## Prerequisites
+- Git ([download](https://git-scm.com/))
+- .NET 5 SDK ([download](https://dotnet.microsoft.com/download/dotnet/5.0))
+- Visual Studio Code ([download](https://code.visualstudio.com/download)) with the following extensions installed:
+  - [C#](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp)
+  - [REST Client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client)
+- Docker for desktop ([download](https://www.docker.com/products/docker-desktop))
+- Dapr CLI and Dapr runtime ([instructions](https://docs.dapr.io/getting-started/install-dapr-selfhost/))
+- Install Azure CLI
+  - Linux ([instructions](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=apt))
+  - macOS ([instructions](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-macos))
+  - Windows ([instructions](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-windows?tabs=azure-cli))
+- Install Bicep extension for VS Code ([instructions](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-bicep))   
+- If you're running Windows, you'll need to install a **bash shell** to run some of the commands. Install either the [Git Bash](https://git-scm.com/downloads) client or the [Windows Subsystem for Linux 2](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+
+## Repository Contents
+- `./Coach/Guides`
```

Comment:
There's no "/Guides" folder.  Just use ./Coach and ./Student.

(And yes... this is the only part of the WTH template that refers to the repo. I'm actually considering removing this section from all hacks in the future.)